fractal/parser.py

```diff
@@ -373,9 +373,17 @@
     argument_default=ap.SUPPRESS,
     allow_abbrev=False,
 )
-task_edit_parser.add_argument(
-    "task_id_or_name", help="ID or name of task to edit", type=str
+
+task_edit_id_or_name_group = task_edit_parser.add_mutually_exclusive_group(
+    required=True
+)
+task_edit_id_or_name_group.add_argument(
```

Comment:
I guess you didn't use `--id` to avoid using the keyword `id` as a function argument.
I propose we keep the `--id` and `--name` argparse arguments, and we set a different `dest` in `add_argument` so that they still point to `task_id` and `task_name` (or to `_id` and `name`, which is also a valid option).

Reply:
To clarify, I expect the command to look like
```
fractal task edit --name SOMETHING
fractal task edit --id 123
```

Reply:
Ref: https://docs.python.org/3/library/argparse.html#dest